tyk-stack/values.yaml

```diff
@@ -1315,3 +1315,8 @@ tyk-dev-portal:
 
   # podLabels specifies labels to be added in Tyk Developer Portal Pod
   podLabels: {}
+
+  # bootstrapPortal contains information that'd be used to bootsrap dev portal
+  bootstrapPortal:
```

Comment:
we also have a feature that allows tyk-k8s-bootstrap script to create a developer portal secret. isn't needed anymore? if so, you might want to disable it

Reply:
okay